src/main/java/core/basesyntax/FlipBit.java

```diff
@@ -9,6 +9,9 @@
      * Договоримся, что биты нумеруются от младшего (индекс 1) к старшему (индекс 32).</p>
      */
     public int flipBit(int value, int bitIndex) {
-        return 0;
+        // reach the bit at the given index by shifting bits
+        // create mask of the number where only predefined bit is set to 1
+        // using XOR replace that bit (XOR returns "true" only if there is one true)
```

Comment:
Good explanation) but don't push comments to GitHub.

Reply:
deal, thank you.